velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -447,10 +514,14 @@ ExpressionFuzzer::ExpressionFuzzer(
     auto returnType = typeToBaseName(it.returnType);
     if (typeToExpressionList_[returnType].empty() ||
         typeToExpressionList_[returnType].back() != it.name) {
-      // Ensure only one entry for a function name is added. This
+      // Ensure entries for a function name are added only once. This
       // gives all others a fair chance to be selected. Since signatures
       // are sorted on the function name this check will always work.
-      typeToExpressionList_[returnType].push_back(it.name);
+      int tickets = getTickets(it.name);
+      // Add multiple entries to increase likelihood of its selection.
+      for (int i = 0; i < tickets; i++) {
+        typeToExpressionList_[returnType].push_back(it.name);
+      }
     }
     expressionToSignature_[it.name][returnType].push_back(&it);
```

Comment:
nit: Not a problem with the current code, but `expressionToSignature_` maps function name to its return type names and further to the callable signatures of the function and return type. On the other hand, the comment in ExpressionFuzzer.h says "Maps the base name of a *concrete* return type signature to the function name...". Should we update the comment to be consistent?

Reply:
Thanks for the nits, I'll address these in a follow up refactor